### PR TITLE
8430 viewer user ui design fixes

### LINKED ISSUE #8430: Viewer user UI design fixes
- [X] Profile: Alignment of elements.
- [X] Dropdown menu: Alignment of elements.
- [X] Add user: Alignment of elements.
- [X] Dropdown (Viewer).

cc @urbanphes

### PR DESCRIPTION
**Pull request by juanignaciosl**
This closes #8430.

@javierarce can you CR this, please?

### CODE REVIEW

File: app/views/admin/organization_users/_form.html.erb [hunk 1]
```diff
@@ -79,11 +79,10 @@
         <label class="RadioButton-label dark-color">Builder (<%= current_user.organization.assigned_seats %>/<%= current_user.organization.seats %>)</label>
       </div>
     </div>
-    <div class="FormAccount-rowInfo u-lspace-28">
+    <div class="FormAccount-rowInfo u-lspace-28 FormAccount-rowLabel">
       <p class="FormAccount-rowInfoText">Build and read</p>
     </div>
-  </div>
-  <div class="FormAccount-row">
+
```
Comment (javierarce) [inline app/views/admin/organization_users/_form.html.erb]: A doubt regarding the removal of the div from line 85: is the divs correctly closed in the HTML?

**Comment by javierarce** [comment]
Except for my comment, the changes are 👍

File: app/views/admin/organization_users/_form.html.erb (hunk 1 above)
Comment (juanignaciosl) [inline app/views/admin/organization_users/_form.html.erb]: Yes. Those were two `FormAccount-row`, one after another, and in order to fix an extra spacing I had to merge both, so the closure of the second now closes the only one. You meant that, isn't it? Deploying after your 👍

Comment (javierarce) [inline app/views/admin/organization_users/_form.html.erb]: Ah, ok!

**Pull request merged**